app/views/course/assessment/answers/_answer.json.jbuilder

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 json.id answer.id
 json.questionId answer.question_id
+json.questionType answer.question.question_type
 json.createdAt answer.created_at&.iso8601
+json.clientVersion answer.client_version
+json.isDraftAnswer answer.draft_answer?
```

Comment:
draft_answer is an attribute of answer not its actable, so I moved it up. Although the initial use case was only for programming, but it could be extended in the future.

Reply:
understand on this part..